setup_chef_server.sh

```diff
@@ -46,6 +46,10 @@ else
   printf "chef_server_webui['enable'] = false\n" >> /etc/chef-server/chef-server.rb
   printf "nginx['enable_non_ssl'] = false\n" >> /etc/chef-server/chef-server.rb
   printf "nginx['non_ssl_port'] = 4000\n" >> /etc/chef-server/chef-server.rb
+  # Configure Solr to index right away when we a new node.  
+  # Reference: https://docs.chef.io/config_rb_server.html#opscode-solr4
+  # Called opscode_solr4 in chef-server 12+
+  printf "chef_solr['max_commit_docs'] = 1\n" >> /etc/chef-server/chef-server.rb
```

Comment:
Nice. [The default](https://docs.chef.io/server_tuning.html#update-frequency) is `1000` or 60 seconds, meaning it's probably always 60 seconds for us.

I think we add objects infrequently enough for this value to make sense, and I can't come up with any downsides at the moment...

![MEME THAT DESCRIBES MY FEELINGS](https://i.imgflip.com/26dwlt.jpg)

Reply:
finished chef-client runs counts as updates to the node objects as well if i understood the docs correctly.

Reply:
How often were you able to reproduce the 500 errors? Does setting this to maybe `10` or `20` alleviate that somewhat? What's the highest value we can set that to where the result will appear in Solr within about 5 seconds or so?

Reply:
Talked with @vt0r to leave as is and see how it fares in our deployments.

The worst case from 500 errors would be chef runs aborting.  We'll look at how chef runs fare ad adjust the interval accordingly.